ui/src/pages/ResultsPage.tsx

```diff
@@ -3915,6 +3984,23 @@ export const SecretsTabContent = (props: {
 										}
 									/>
 								</ListItem>
+								<ListItem key="secret-validity">
+									<ListItemText
+										primary={
+											<>
+												{i18n._(t`Validity`)}
```

Comment:
Since this is in a JSX element and not an attribute of an element, I think we could also use `<Trans> Validity</Trans>` here, but if this works and follows the pattern in other places of the code, leave as-is